Dockerfile.dapper

```diff
@@ -40,17 +43,30 @@ ENV HELM_URL_amd64=https://github.com/rancher/helm/releases/download/${CATTLE_HE
     K3S_URL=K3S_URL_${ARCH} \
     ETCD_URL_amd64=https://github.com/etcd-io/etcd/releases/download/${CATTLE_ETCD_VERSION}/etcd-${CATTLE_ETCD_VERSION}-linux-amd64.tar.gz \
     ETCD_URL_arm64=https://github.com/etcd-io/etcd/releases/download/${CATTLE_ETCD_VERSION}/etcd-${CATTLE_ETCD_VERSION}-linux-arm64.tar.gz \
-    ETCD_URL=ETCD_URL_${ARCH}
+    ETCD_URL=ETCD_URL_${ARCH} \
+    KUSTOMIZE_URL_amd64=https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_amd64.tar.gz \
+    KUSTOMIZE_URL_arm64=https://github.com/brendarearden/kustomize/releases/download/kustomize/${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_arm64.tar.gz \
+    KUSTOMIZE_URL=KUSTOMIZE_URL_${ARCH}
 
-RUN curl -sLf ${!HELM_URL} > /usr/bin/rancher-helm && \
+RUN if [ "${ARCH}" == "arm64" ]; then \
```

Comment:
Nit: would be nice to have little one-line comments for these blocks of code that say: `Fetch kustomize for given arch`. Next block can be something like `Setup helm v2/v3, tiller and k3s`. Not necessary but its nice for skimming.

Reply:
Does this get fixed with the move to our own kustomize release? Meaning same code for both architectures

Reply:
yes, the arch code will go away